packages/components/modal/src/modal.js

```diff
@@ -38,6 +40,15 @@ customElementDefineHelper(
 			return this.visible ? 'fade-in' : 'fade-out';
 		}
 
+		get header() {
+			if (this.hideHeader) return '';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)